### DISCUSSION TITLE
Vercel does not show images with external link

### DISCUSSION THREAD

**Discussion by JoergWolff** [neuefische/questions#459] [Web] [answered]
Hello dear fish, I have the problem that in my app the images are displayed locally, but not in Vercel. I have a JSON object in which an external link points to the image.
export const initialData = [
  {
    id: 1,
    isActive: true,
    type: "book",
    author: "Stephen King",
    title: "Feuerkind",
    subtitle: "",
    coverImg: "https://covers.openlibrary.org/b/id/10137891-L.jpg",
    isbn10: "3-453-43273-8",
    teaser:
      "Bla bla",
  },]

But [At vercel](https://capstone-project-2tkiliew1-joergwolff.vercel.app/)
Is there still a setting at Vercel?
Thanks for now

**Comment by doemser** [comment, marked as answer]
It looks like you still need to add the links to external images in your next.config.js.

```javascript
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  images: {
    domains: ["avatars.githubusercontent.com"],
  },
};

module.exports = nextConfig;
````
Take a look at next.js Docs starting here:

https://nextjs.org/docs/api-reference/next/image#remote-patterns

The approach I used in my example is based on this:

https://nextjs.org/docs/api-reference/next/image#domains

Did that already help you?

**Comment by JoergWolff** [comment]
Thanks for the quick reply. You are right. My mistake was that I entered it in the main only after I had created my branch. Of course, the entry was not present in the branch.
